dashboard/src/components/Catalog/Catalog.tsx

```diff
@@ -483,6 +493,19 @@ export default function Catalog() {
                   />
                 </div>
               )}
+              {allPlugins.length > 0 && (
+                <div className="filter-section">
+                  <label className="filter-label">Package Type</label>
+                  <FilterGroup
+                    name={filterNames.PKG_TYPE}
+                    options={allPlugins}
```

Comment:
Perhaps we should differentiate the plugin name from the package type label. With this, we are listing a friendly name of active plugins, e.g.: 

- Helm
- Kapp
- Flux

But the control is for `Package Type`. There is no "Flux" packaging format per se, it is a Helm chart.
So packages types would be slightly different from plugin names:
- Helm
- Kapp (or Carvel)
- Helm via Flux

WDYT?

Reply:
Good idea. Initially I used "Packaging Plugin" as the filter name, hence the names I chose. But, 100% agree, now we're using "Package Type", we better use this existing function:

https://github.com/vmware-tanzu/kubeapps/blob/88030f61605dab0fd176ae14d4ef22b2b29438a6/dashboard/src/shared/utils.ts#L125-L133

See the result:

![image](https://user-images.githubusercontent.com/11535726/188623445-6f407705-bab0-450a-9bd4-9f42ead73f7d.png)